features/aanhef.feature

```diff
@@ -2,216 +2,749 @@
-      | man      | man      | Baron    | Partner na eigen      | W. van der Veen-van den Aedel           | Geachte heer Van der Veen-van den Aedel    |
-      | onbekend | man      | Baron    | Partner na eigen      | W. van der Veen-van den Aedel           | Geachte W. van der Veen-van den Aedel      |
-      | vrouw    | vrouw    | Barones  | Partner na eigen      | W. van der Veen-van den Aedel           | Geachte mevrouw Van der Veen-van den Aedel |
-      | vrouw    | man      | Ridder   | Partner na eigen      | W. van der Veen-van den Aedel           | Geachte mevrouw Van der Veen-van den Aedel |
-      | vrouw    | man      | Ridder   | Partner               | W. van den Aedel                        | Geachte mevrouw Van den Aedel              |
-      | vrouw    | man      | Jonkheer | Eigen                 | A.C. van der Veen                       | Geachte mevrouw Van der Veen               |
-      | vrouw    | man      | Jonkheer | Partner na eigen      | A.C. van der Veen-van den Aedel         | Geachte mevrouw Van der Veen-van den Aedel |
-      | vrouw    | man      | Jonkheer | Partner               | A.C. van den Aedel                      | Geachte mevrouw Van den Aedel              |
-      | vrouw    | man      | Jonkheer | Partner voor eigen    | A.C. van den Aedel-van der Veen         | Geachte mevrouw Van den Aedel-van der Veen |
-
-  Scenario: meerdere actuele relaties
-    Gegeven de ingeschreven persoon de heer F.C. Groen is getrouwd in 1958 met Geel
-    En de ingeschreven persoon is getrouwd in 1961 met Roodt
-    En geen van beide relaties is beëindigd
-    En de ingeschreven persoon heeft aanduidingAanschrijving='V'
-    Als de ingeschreven persoon wordt geraadpleegd
-    Dan is in het antwoord naam.aanhef=Geachte heer Geel-Groen
-
-  Scenario: meerdere ontbonden relaties gebruikt de laatst ontbonden relatie
-    Gegeven de ingeschreven persoon de heer J. Wit is getrouwd in 1958 met Geel
-    En de ingeschreven persoon is getrouwd in 1961 met Roodt
-    En het huwelijk met Geel is ontbonden in 1960
-    En het huwelijk met Roodt is ontbonden in 2006
-    En de ingeschreven persoon heeft aanduidingAanschrijving='V'
-    Als de ingeschreven persoon wordt geraadpleegd
-    Dan is in het antwoord naam.aanhef=Geachte heer Roodt-Wit
-
-
-  Scenario: meerdere ontbonden relaties en oudste relatie is het laatst ontbonden
-    Gegeven de ingeschreven persoon de heer J. Wit is getrouwd in 1958 met Zwart
-    En de ingeschreven persoon is getrouwd in 1961 met Blaauw
-    En het huwelijk met Blaauw is ontbonden in 1983
-    En het huwelijk met Zwart is ontbonden in 2006
-    En de ingeschreven persoon heeft aanduidingAanschrijving='V'
-    Als de ingeschreven persoon wordt geraadpleegd
-    Dan is in het antwoord naam.aanhef=Geachte heer Zwart-Wit
+  De aanhef bij een persoon wordt gevuld door de provider om op deze wijze op eenduidige wijze een persoon te kunnen aanschrijven.
```

Comment:
kan deze niet ook worden verwijderd? We hebben toch ook bevragen/persoon/adressering/aanhef?